package.json

```diff
@@ -18,11 +18,16 @@
     "lint": "eslint 'src/**/*.js'",
     "prettier": "prettier --write '{src/**/,scripts/**/,}*.js'",
     "prettier:check": "prettier-check '{src/**/,scripts/**/,}*.js'",
-    "test": "jest --no-cache --coverage",
+    "test": "jest --no-cache --coverage src",
     "test:ci": "run-p test prettier:check lint",
+    "test:watch": "jest --watch --runInBand src",
+    "test:watch:focus": "jest --watch --runInBand --coverage --coverageReporters=text --collectCoverageFrom=\"${FOCUS}/**/*.js\" --no-cache \"${FOCUS}.*\"",
```

Comment:
Do we truly need these? I expect people to be running `npm run` to see a list of commands to use, and the more we have, the harder it's going to be for them to decipher. These seem like things that may be better for your personal workspace (I wouldn't be using them).

Reply:
Sure, I can remove them. I like them though; they help me use the same filter on tests and coverage measurements, so I can work towards 100% instead of a screen full of red garbage from covering the tests that didn't run. Is there a simpler way to do that?